.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           apt-get update
           apt-get install -y python-pip
           pip install flake8==3.4.1
-          flake8 tools/response_file.py tools/wasm-sourcemap.py
```

Comment:
I assume `.flake8_clean` is a text file with a list of files to flake8
But I don't see it on the diff :p